1ejer/index.css

```diff
@@ -0,0 +1,111 @@
+.container{
```

Comment:
Mejor ponerle algún nombre específico de la página.
```suggestion
.ejer1-container {
```